packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte

```diff
@@ -122,12 +125,22 @@
         <div class="relative w-full h-full flex rounded-2xl overflow-hidden">
             <NftMedia
                 nftId={id}
+                bind:error
+                bind:warning
                 classes="rounded-2xl overflow-hidden flex-1 w-auto h-auto max-w-full max-h-full object-contain absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2"
                 autoplay
                 controls
                 loop
                 muted
+                big
             />
+            <div class="toast-container">
```

Comment:
Why create an scss class instead of using tailwind:
`absolute right-6 bottom-6 w-auto`